docs/hardening.md

```diff
@@ -0,0 +1,112 @@
+# Cluster Hardening
+
+If you want to improve the security on your cluster and make it compliant with the [CIS Benchmarks](https://learn.cisecurity.org/benchmarks), here you can find a configuration to harden your **Kubernetes** installation.
+
+To apply the hardening configuration, create a file (eg. `hardening.yaml`) and paste the content of the following code snippet into that.
+
+**N.B.** Some of these configurations have just been added to **Kubespray**, so ensure that you have the latest version to make it works properly.
+
+Also, ensure that other configurations doesn't override these.
+
+`hardening.yaml`:
+
+```yaml
+# Hardening
+---
+# This allow us to use PodSecurity admission plugin which is provided as built-in
+kube_version: v1.23.6
+
+## kube-apiserver
+authorization_modes: ['Node','RBAC']
+kube_apiserver_feature_gates: ['AppArmor=true']
```

Comment:
I guess this configuration is not enabled for RHEL/CentOS, right?

Reply:
My experience using **kubespray** is with Debian/Ubuntu OS. Maybe I can write a note about that on top of the document to specify that it has been tested with these OS only. WDYT?

Reply:
> We can remove this and leave a comment as suggestion to enable it if you are using an AppArmor based OS.

That seems good for me.
I just didn't want to see some issues like `this hardening way doesn't work on my environment` from who are using RHEL OS :sweat_smile: